handshake_messages_test.go

```diff
@@ -172,6 +172,8 @@ func (*clientHelloMsg) Generate(rand *rand.Rand, size int) reflect.Value {
 		m.earlyData = true
 	}
 
+	m.delegatedCredential = true
```

Comment:
So, this is wrong - delegatedCredentials and EMS should be set randomly to true/false